syncmaster/backend/scripts/manage_superusers.py

```diff
@@ -0,0 +1,105 @@
+#!/bin/env python3
+
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import argparse
+import asyncio
+import logging
+
+from sqlalchemy.ext.asyncio import AsyncSession, async_sessionmaker, create_async_engine
+from sqlalchemy.future import select
+
+from syncmaster.backend.middlewares import setup_logging
+from syncmaster.backend.settings import BackendSettings as Settings
+from syncmaster.db.models.user import User
+
+
+async def add_superusers(session: AsyncSession, usernames: list[str]) -> None:
+    logging.info("Adding superusers:")
+    result = await session.execute(select(User).where(User.username.in_(usernames)).order_by(User.username))
+    users = result.scalars().all()
+
+    not_found = set(usernames)
+    for user in users:
+        user.is_superuser = True
+        logging.info("    %r", user.username)
+        not_found.discard(user.username)
+
+    if not_found:
+        for username in not_found:
+            session.add(User(username=username, email=f"{username}@mts.ru", is_active=True, is_superuser=True))
+            logging.info("    %r (new user)", username)
+
+    await session.commit()
+    logging.info("Done.")
+
+
+async def remove_superusers(session: AsyncSession, usernames: list[str]) -> None:
+    logging.info("Removing superusers:")
+    result = await session.execute(select(User).where(User.username.in_(usernames)).order_by(User.username))
+    users = result.scalars().all()
+
+    not_found = set(usernames)
+    for user in users:
+        logging.info("    %r", user.username)
+        user.is_superuser = False
+        not_found.discard(user.username)
+
+    if not_found:
+        logging.info("Not found:")
+        for username in not_found:
+            logging.info("    %r", username)
+
+    await session.commit()
+    logging.info("Done.")
+
+
+async def list_superusers(session: AsyncSession) -> None:
+    result = await session.execute(select(User).filter_by(is_superuser=True).order_by(User.username))
+    superusers = result.scalars().all()
+    logging.info("Listing users with SUPERUSER role:")
+    for superuser in superusers:
+        logging.info("    %r", superuser.username)
+    logging.info("Done.")
+
+
+def create_parser() -> argparse.ArgumentParser:
```

Comment:
Should we add this script usage example to documentation, or this is a part of some dedicated PR?

Reply:
Added usage example for docker, we can also extend script usage later